src/com/amazon/ion/impl/lite/IonStructLite.java

```diff
@@ -460,7 +460,7 @@ public boolean add(IonValue child)
 
     public ValueFactory add(final String fieldName)
     {
-        return new _Private_CurriedValueFactory(_context.getSystem())
+        return new _Private_CurriedValueFactory(getSystem())
```

Comment:
This was harmless before--`getSystem()` was the same as calling `_context.getSystem()`. This is no longer the case.